script/binary-cli

```diff
@@ -16,8 +16,8 @@
 
 set -e
 
-OS_PLATFORM_ARG=(-os="darwin linux windows")
```

Comment:
you don't want the CLI on OSX or Windows ?

Reply:
true, completely missed that